jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -98,38 +95,21 @@ def process(self, input, *args, **kwargs):
         if not (result.meta.exposure.type.lower() in IMAGING_TYPES and self.sip_approx):
             return result
 
+        (result, )
+
         # fit sip approx., degree is chosen by best fit
-        sip_degree = range(1, _MAX_SIP_DEGREE) if self.sip_degree is None else self.sip_degree
-        sip_inv_degree = range(1, _MAX_SIP_DEGREE) if self.sip_inv_degree is None else self.sip_inv_degree
-        crpix = [result.meta.wcsinfo.crpix1, result.meta.wcsinfo.crpix2]
-        crpix = None if None in crpix else crpix
         try:
```

Comment:
maybe add a logging message that the fits header is being updated (and that this wcs should be considered an approximation)

Reply:
I do not think this is necessary. I am worried about unnecessary warnings. The code is doing what it is designed to do and what people expect it to do.

Reply:
I agree with @mcara. We're already putting FITS SIP in cal files, without warning anyone, so I don't think a warning is needed here either.